### ISSUE TITLE
PSR-7 adapter for Amphp HTTP Server

### ISSUE THREAD

**Issue by raghuveer** [amphp/http-server#365]
> We plan to eventually have a PSR-7 adpter on top of our current API, but we don't plan to make our API directly depend on PSR-7 here.

it will be really nice to have PSR-7 support through PSR-7 adapter

_Originally posted by @raghuveer in https://github.com/amphp/http-server/issues/357#issuecomment-1925322985_

**Comment by raghuveer** [comment]
@kelunik can this request be prioritized? is it possible?

any email address/chat option for a discussion on this and for prioritization of few other custom requirements?
